src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
toString is called automatically inside System.out.println()